src/httpserver.cpp

```diff
@@ -487,6 +487,10 @@ void StopHTTPServer()
     if (workQueue) {
         LogPrint(BCLog::HTTP, "Waiting for HTTP worker threads to exit\n");
         workQueue->WaitExit();
```

Comment:
I think we don't need WaitExit() anymore with this?

Reply:
It would seem not, indeed. Testing shows no difference with it removed.

Reply:
Ok, will remove it in a separate commit.